.circleci/config.yml

```diff
@@ -102,6 +102,7 @@ jobs:
 
 # integration
   integration-tests:
+    resource_class: xlarge
```

Comment:
Did we fail without the larger class size here? Why the increase resource usage?